internal/service/networkfirewall/firewall_policy_test.go

```diff
@@ -328,6 +328,34 @@ func TestAccNetworkFirewallFirewallPolicy_statefulRuleGroupPriorityReference(t *
 	})
 }
 
+func TestAccNetworkFirewallFirewallPolicy_statefulRuleGroupOverrideActionReference(t *testing.T) {
+	var firewallPolicy networkfirewall.DescribeFirewallPolicyOutput
+	rName := sdkacctest.RandomWithPrefix(acctest.ResourcePrefix)
+	resourceName := "aws_networkfirewall_firewall_policy.test"
+	override_action := "DROP_TO_ALERT"
```

Comment:
Great reference to the valid value documentation for `DROP_TO_ALERT`. As an enhancement, instead of hardcoding this string, we can reference this string using `networkfirewall.OverrideActionDropToAlert`

```suggestion
	override_action := networkfirewall.OverrideActionDropToAlert

Reply:
Updated.